Subject: Linux-RT 2.6.27-RT
From: http://www.kernel.org/pub/linux/kernel/projects/rt/
Acked-by: Sven-Thorsten Dietrich <sdietrich@suse.de>
Subject: Linux-RT 2.6.26-RT
From: http://www.kernel.org/pub/linux/kernel/projects/rt/
Acked-by: Sven-Thorsten Dietrich <sdietrich@suse.de>
From: Steven Rostedt <srostedt@redhat.com>
Subject: rt: remove call to stop tracer

Remove user_trace_stop that was more of a hack to debug xrun latencies.

Signed-off-by: Steven Rostedt <srostedt@redhat.com>
---
 sound/core/pcm_lib.c |    1 -
 1 file changed, 1 deletion(-)

Index: linux-2.6.26/sound/core/pcm_lib.c
===================================================================
--- linux-2.6.26.orig/sound/core/pcm_lib.c
+++ linux-2.6.26/sound/core/pcm_lib.c
@@ -130,7 +130,6 @@ static void xrun(struct snd_pcm_substrea
 	snd_pcm_stop(substream, SNDRV_PCM_STATE_XRUN);
 #ifdef CONFIG_SND_PCM_XRUN_DEBUG
 	if (substream->pstr->xrun_debug) {
-		user_trace_stop();
 		snd_printd(KERN_DEBUG "XRUN: pcmC%dD%d%c\n",
 			   substream->pcm->card->number,
 			   substream->pcm->device,
